lib/data_migrate/database_tasks.rb

```diff
@@ -47,6 +47,40 @@ def check_schema_file(filename)
           Kernel.abort message
         end
       end
+
+      def pending_migrations
+        sort_migrations(
+          pending_schema_migrations,
+          pending_data_migrations
+        )
+      end
+
+      def sort_migrations set_1, set_2=nil
+        migrations = set_1 + (set_2 || [])
+        migrations.sort{|a,b|  sort_string(a) <=> sort_string(b)}
+      end
+
+      def sort_string migration
+        "#{migration[:version]}_#{migration[:kind] == :data ? 1 : 0}"
+      end
+
+      def data_migrations_path
+        ::DataMigrate.config.data_migrations_path
+      end
+
+      def run_migration(migration, direction)
+        if migration[:kind] == :data
+          ::ActiveRecord::Migration.write("== %s %s" % ['Data', "=" * 71])
+          ::DataMigrate::DataMigrator.run(direction, data_migrations_path, migration[:version])
+        else
+          ::ActiveRecord::Migration.write("== %s %s" % ['Schema', "=" * 69])
+          ::DataMigrate::SchemaMigration.run(
+            direction,
+            ::Continuation::SchemaMigration.migrations_paths,
```

Comment:
Hi @ilyakatz, I'm not sure why I ended up with this weird class name here 🤣 

This could be `DataMigrate::SchemaMigration.migrations_paths` supposedly FYI?

Reply:
mm, yeah, does this actually break stuff or just naming? want to submit a quick PR?

Reply:
Yeah, I think that's the issue here https://github.com/ilyakatz/data-migrate/pull/218#issuecomment-1202145878

Here you are: https://github.com/ilyakatz/data-migrate/pull/236